packages/block-editor/src/components/block-mobile-toolbar/index.native.js

```diff
@@ -6,50 +6,60 @@ import { Keyboard, View } from 'react-native';
 /**
  * WordPress dependencies
  */
-import { ToolbarButton } from '@wordpress/components';
-import { __, sprintf } from '@wordpress/i18n';
 import { withDispatch, withSelect } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
-import { trash } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 import BlockMover from '../block-mover';
+import BlockActionsMenu from './block-actions-menu';
 import { BlockSettingsButton } from '../block-settings';
 
+const BREAKPOINTS = {
+	wrapSettings: 65,
+	wrapMover: 150,
+};
 const BlockMobileToolbar = ( {
 	clientId,
 	onDelete,
-	order,
 	isStackedHorizontally,
-} ) => (
-	<View style={ styles.toolbar }>
-		<BlockMover
-			clientIds={ [ clientId ] }
-			isStackedHorizontally={ isStackedHorizontally }
-		/>
+	blockWidth,
+	blockMobileToolbarRef,
```

Comment:
Hmmm, wdyt about a different name for this prop? It's a bit confusing that we pass `blockMobileToolbarRef` to the `block-mobile-toolbar` component

Reply:
I think the name is correct and descriptive - the ref is related to `blockMobileToolbar`. Any suggestion how can I rename it?

Reply:
As far as i can see it is rather a ref to the view that contains the block-mobile-toolbar. I agree that the ref is related to the `blockMobileToolbar` but this is kinda weird that we pass `blockMobileToolbarRef` to the `blockMobileToolbar`, don't you think? The name suggest that this ref points to the `BlockMobileToolbar` component but it doesn't.

Reply:
I've renamed it to `blockMobileToolbarWrapperRef`, however, it looks too long imo.

Reply:
Hmmm. yeah, I agree that it's too long. WDYT about something like `anchorNodeRef` or `actionsMenuAnchor`?

Reply:
`anchorNodeRef` makes sense!